crates/turbopack-node/src/render/render_proxy.rs

```diff
@@ -258,9 +262,17 @@ async fn render_stream_internal(
         }
         operation.send(RenderProxyOutgoingMessage::BodyEnd).await?;
 
+        let entry = module.ident().to_string().await?;
+        let node_execution_start = Instant::now();
+        let end_execution = || {
```

Comment:
This would be more elegantly represented by a span guard. ```let node_execution_span = duration_span!(Level::TRACE, "Node.js rendering");``` to be `drop`ped later.

You're reusing this pattern multiple times above and below, so it makes sense to abstract it.